package.json

```diff
@@ -41,24 +41,24 @@
     "browserstack-cypress-cli": "^1.10.1",
     "cli-table": "^0.3.6",
     "codecov": "^3.8.3",
-    "concurrently": "^6.3.0",
+    "concurrently": "^6.4.0",
     "cypress": "7.2.0",
-    "es-check": "^6.0.0",
+    "es-check": "6.0.0",
```

Comment:
This being pinned, as there is a potential issue with `6.1.0`, have raised it: https://github.com/yowainwright/es-check/issues/139